pyproject.toml

```diff
@@ -15,7 +15,7 @@ keywords = [
     "model arch",
     "model architecture",
 ]
-license = { text = "GPLv3" }
+license = "MIT"
```

Comment:
This needs the text field back

Reply:
Why? I looked up examples and they didn't use the `text` field.

Reply:
Oh, I guess not. That's what I thought the build error was for, but I guess it just doesnt recognize MIT?

Reply:
Oh, I didn't even look at the errors yet. Hmmm, this might a difference between tooling then. I add the `text` field again.